### DISCUSSION TITLE
Can Sigma File Manager be set as the default?

### DISCUSSION THREAD

**Discussion by SongZihui-sudo** [aleksey-hoffman/sigma-file-manager#234] [Q&A]
Can Sigma File Manager be set as the default?

**Comment by aleksey-hoffman** [comment]
Currently not, but I will add that feature later

**Reply by SongZihui-sudo** [reply]
OK,🤗

**Reply by roke-julian-lockhart** [reply]
@SongZihui-sudo, https://github.com/aleksey-hoffman/sigma-file-manager/issues/218#issuecomment-1471535723
